rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -2106,19 +2106,63 @@ namespace {
 	}
 }
 
+/***
+  * Drawing Flag
+  *
+  * @alias DrawFlag
+  * | 0 # No draw
+  * | 1 # Opaque
+  * | 2 # Alpha F
+  * | 4 # Reflection
+  * | 8 # Refraction
+  * | 16 # Sh Opaque
+  * | 32 # Sh Transparent
+  * | 128 # Icon
+  */
+
+/***
+  * Bitwise mask of `DrawFlag`
+  *
+  * @see DrawFlag
+  * @alias DrawMask integer
+  */
+
+/***
+ *
+ * @function Spring.GetRenderUnits
+ * @param drawMask DrawMask (Default: `0`) Filter objects by their draw flags.
+ * @param sendMask true Whether to send objects draw flags as second return
+ * @return integer[] featureIDs
+ * @return DrawFlag[] drawFlags
+ */
 
 /***
  *
  * @function Spring.GetRenderUnits
+ * @param drawMask DrawMask (Default: `0`) Filter objects by their draw flags.
+ * @param sendMask false? Whether to send objects draw flags as second return
+ * @return integer[] featureIDs
+ * @return nil
```

Comment:
Can remove

Reply:
Do you mean we don't need to make `@return nil` explicit? This was ubiquitous in the existing docs so I have retained them. I agree that implicit `nil` return is enough.

Reply:
Since we've overloaded, yes